apps/src/applab/applab.js

```diff
@@ -357,16 +356,13 @@ function handleExecutionError(err, lineNumber) {
   if (!lineNumber && err instanceof SyntaxError) {
     // syntax errors came before execution (during parsing), so we need
     // to determine the proper line number by looking at the exception
-    lineNumber = err.loc.line - Applab.userCodeLineOffset;
+    lineNumber = err.loc.line;
```

Comment:
We used to insert a "prologue" of functions that wrapped our APIs (e.g. `function moveForward() { return Applab.moveForward(); }`). I removed that a few weeks ago by inserting the APIs into the interpreter directly as global functions. I don't think we'll add back a prologue, so I've started removing the code that accounted for it. We do still have the "epilogue" (currently, a mini runtime that processes events in a loop after the main function is complete).